### ISSUE TITLE
[Search-bar] The input placeholder is shown even if the query is not empty

### ISSUE THREAD

**Issue by JoanaDalmeida** [KleeGroup/focus-components#1137]
# Current behaviour

Even if there is a 'query' the placeholder is shown.

![image](https://cloud.githubusercontent.com/assets/3300779/14990486/1193f8fc-115d-11e6-8a17-5b151da3f3c8.png)

# Expected behaviour

Hide the placeholder if there is a query.

## Versions

Focus-core: 0.14.0
Focus-components: 0.14.1

**Comment by pierr** [comment]
@GuenoleK Is it a material problem?

**Comment by GuenoleK** [comment]
Yes it's a material design problem @JoanaDalmeida & @pierr . The placeholder is not updated.
We  should update input/text component this way :

**Comment by TomGallon** [comment]
waiting to be merged in #1144

**Comment by TomGallon** [comment]
@pierr @GuenoleK what do we do ????

**Comment by GuenoleK** [comment]
Cannot reproduce the bug
